src/app/wallets/update-pending-invoices.ts

```diff
@@ -80,6 +84,24 @@ export const updatePendingInvoiceByPaymentHash = async ({
   return updatePendingInvoice({ walletInvoice, logger, lock })
 }
 
+const usdHedgeEnabled = getDealerConfig().usd.hedgingEnabled
+const dealer = NewDealerPriceService()
+
+const usdFromBtcMidPriceFn = async (
+  amount: BtcPaymentAmount,
+): Promise<UsdPaymentAmount | DealerPriceServiceError> => {
+  // TODO: Decide if we should fallback to PriceService if Dealer fails; remove TODO if not
```

Comment:
This TODO has been resolved or?

Reply:
Resolved here (https://github.com/GaloyMoney/galoy/pull/1237/commits/694b4a7ab3c5c433b2fffb8fa148702e360912c0)